spec/services/reports/region_service_spec.rb

```diff
@@ -35,6 +41,224 @@ def refresh_views
     end
   end
 
+  it "counts registrations and cumulative registrations" do
```

Comment:
All these specs came over from ControlRateService, and I think its worth keeping them around for now just for a bit more confidence in the legacy things here.